src/Microsoft.Extensions.CodeGeneration/CodeGenCommand.cs

```diff
@@ -5,42 +5,42 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
-using System.Threading.Tasks;
-using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.Extensions.CodeGeneration
 {
     public class CodeGenCommand
     {
+        private readonly ILogger _logger;
+        private readonly ICodeGeneratorLocator _locator;
+
```

Comment:
We probably could get rid of the IServiceProvider dependency now, but not sure if that's needed for consumer usage outside of this class